Makefile

```diff
@@ -60,5 +60,8 @@ $(LINTERS): %: vendor/bin/gometalinter %-bin vendor
 	PATH=`pwd`/vendor/bin:$$PATH gometalinter --tests --disable-all --vendor \
 		--deadline=5m -s data --enable $@ ./...
 
+cover: vendor
```

Comment:
I'd change the `ci` target to use `cover` instead of `test` so tests with coverage get run on CI